### PR TITLE
feat(Sharing): qr-code

### PR DESCRIPTION
**Pull request by roiLeo**
- Changed Sharing component on: Collection View; Profile View; GalleryItem View (move sharing on top before price)
- replace icon "share-alt" to "share"

### PR type
- [x] Feature 
- [x] #1065 related
- [x] #1012 related

### Before submitting Pull Request, please make sure:
- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted screenshot of demonstrated change in this PR

### Screenshot
Screenshot 2021-11-10 at 18-22-07 Kitty Paradise

Screenshot 2021-11-10 at 18-22-01 Kitty Paradise

Screenshot 2021-11-10 at 18-21-50 Kitty Paradise

Screenshot 2021-11-10 at 18-23-18 Kitty Paradise

Screenshot 2021-11-09 at 14-25-37 Kitty Paradise #160

### Todo
- [x] responsive Sharing buttons
- [x] column spacing
- [x] modal header
- [x] gift button
- [x] fix responsive tablet/mobile?

### CODE REVIEW

**Comment by yangwao** [comment]
If we can have this merged before Dotsama Vienna and Dotsama Brno, that could be amazing 👀
Happens in 1-2 week iirc

**Comment by yangwao** [comment]
On collection if we can put it aligned?
https://deploy-preview-1063--nftkodadot.netlify.app/rmrk/collection/c87323333f9ce3e831-AIART

image

**Comment by roiLeo** [comment]
> On collection if we can put it aligned? https://deploy-preview-1063--nftkodadot.netlify.app/rmrk/collection/c87323333f9ce3e831-AIART
> 
> image

I'd like to replace DonationButton text by "GIFT" instead of "GIFT 10$" as it will better for me & #939, what do you think?

**Comment by yangwao** [comment]
> > On collection if we can put it aligned? https://deploy-preview-1063--nftkodadot.netlify.app/rmrk/collection/c87323333f9ce3e831-AIART
> > image
> 
> I'd like to replace DonationButton text by "GIFT" instead of "GIFT 10$" as it will better for me & #939, what do you think?
> 
> _With_ Screenshot 2021-11-10 at 11-41-16 Machine ArtWork
> 
> _Without_ Screenshot 2021-11-10 at 11-42-58 Machine ArtWork

oh sure, let's do it without.

**Comment by yangwao** [comment]
OR! let's go with slots like @vikiival suggested and do Gift only symbol, w/o text.

image

**Comment by roiLeo** [comment]
> OR! let's go with slots like @vikiival suggested and do Gift only symbol, w/o text.
> 
> image

Screenshot 2021-11-10 at 17-20-49 Kitty Paradise

**Comment by yangwao** [comment]
> ? Screenshot 2021-11-10 at 17-20-49 Kitty Paradise

Let's put GIFT on the start so it's closer to QR as both has "rectangle" like shapes and feel free to merge it! :)

File: src/components/rmrk/Gallery/Item/Sharing.vue [hunk 1]
```diff
@@ -1,7 +1,12 @@
 <template>
   <div class="share">
     <b-field position="is-right">
+      <div>
+        <slot />
```
Comment (vikiival) [inline src/components/rmrk/Gallery/Item/Sharing.vue:5]: if you put just slot here (omit `<div>` ) does it break the layout?

Comment (roiLeo) [reply src/components/rmrk/Gallery/Item/Sharing.vue:5]: `<div>` is used here to remove spacing between button

**Comment by yangwao** [comment]
> Screenshot 2021-11-10 at 18-22-07 Kitty Paradise

Is it possible to remove double borders/walls? 😄

**Comment by yangwao** [comment]
LGTM 
image

**Pull request merged**